src/commands/math/advancedSymbols.js

```diff
@@ -239,8 +239,42 @@ LatexCmds.bull = LatexCmds.bullet = bind(VanillaSymbol,'\\bullet ','&bull;');
 LatexCmds.setminus = LatexCmds.smallsetminus =
   bind(VanillaSymbol,'\\setminus ','&#8726;');
 
-LatexCmds.not = //bind(Symbol,'\\not ','<span class="not">/</span>');
 LatexCmds['¬'] = LatexCmds.neg = bind(VanillaSymbol,'\\neg ','&not;');
+LatexCmds.not = P(VanillaSymbol, function(_, super_) {
+  // If one of these appears immediately after not, the
+  // parser returns a different symbol.
+  _.suffixes = {
+    '\\ni':       'notni',
+    '\\subset':   'notsubset',
+    '\\subseteq': 'notsubseteq',
+    '\\supset':   'notsupset',
+    '\\supseteq': 'notsupseteq'
+  };
+  _.init = function() {
+    return super_.init.call(this, '\\neg ', '&not;');
+  };
+  _.parser = function() {
+    var succeed = Parser.succeed;
+    var optWhitespace = Parser.optWhitespace;
+
+    // Sort the suffixes, longest first
+    var suffixes = Object.keys(_.suffixes).sort(function(a, b) {
```

Comment:
Should I maybe cache all this stuff (including the `anyOf` parser) once during `_.init` rather than doing this work every time `_.parser` is called? I left it here because (a) I didn't see anywhere else where such a specific parser is cached, and (b) I like the idea of being able to change the suffixes dynamically (although this isn't happening at present).